src/components/daily-dropoffs-report.tsx

```diff
@@ -51,6 +51,8 @@ function tabulate({
       "white" as unknown as number,
     ]);
 
+  const totals = funnelSteps(funnelMode).map(() => 0);
```

Comment:
Good use case for `Array#fill`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill